app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) == 3 and command_parts[0] == "mv":
+        source_file = command_parts[1]
+        new_file = command_parts[2]
```

Comment:
For more descriptive code, perform unpacking in the future instead of indexing for value retrieval